archetypes/marketplace.md

```diff
@@ -0,0 +1,69 @@
+---
+slug: {{ path.Base .File.Dir }}
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'Two to three sentences describing your guide.'
+og_description: 'Two to three sentences describing your guide when shared on social media.'
+keywords: ['list','of','keywords','and key phrases']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: {{ now.Format "2006-01-02" }}
+modified_by:
+  name: Linode
+title: "{{ replace (path.Base .File.Dir) "-" " " | title }}"
+h1_title: "h1 title displayed in the guide."
+contributor:
+  name: Your Name
+  link: Github/Twitter Link
+external_resources:
+- '[Link Title 1](http://www.example.com)'
+- '[Link Title 2](http://www.example.net)'
+---
+
+## ______App Name______ Marketplace App
+
+<!-- Intro paragraph describing the app and what it accomplishes. -->
+
+### Deploy a Guacamole Marketplace App
```

Comment:
Should we substitute Guacamole for ______APP NAME_____ here and on a few other lines it appears? @andystevensname